bin/check-partition-usage.rb

```diff
@@ -0,0 +1,212 @@
+#! /usr/bin/env ruby
+#
+#   check-disk
+#
+# DESCRIPTION:
+#   Uses the sys-filesystem gem to get filesystem mount points disk usage
+#   --mounts '/:w75:c90,/boot:w80:c95'
+#   mounts are comma-separated
+#   warning threshold must be preceeded by the letter w and comes in the second position
+#   critical threshold must be preceeded by the letter c and comes in the third position
+#
+# OUTPUT:
+#   plain text
+#
+# PLATFORMS:
+#   Linux, BSD, Windows
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   gem: sys-filesystem
+#
+# USAGE:
+#   check-partition-usage.rb --help
+# NOTES:
+#
+# LICENSE:
+#   Copyright 2016 Magic Online - www.magic.fr
+#   Heavily inspired from check-disk-usage
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'sys/filesystem'
+include Sys
+
+#
+# Check Disk Partition Usage
+#
+class CheckDiskPartitionUsage < Sensu::Plugin::Check::CLI
+  option(
```

Comment:
I don't see any value in having the parenthesis here and it makes it inconsistent with the other checks.